bitnami/kube-prometheus/Chart.yaml

```diff
@@ -32,4 +32,4 @@ maintainers:
 name: kube-prometheus
 sources:
   - https://github.com/bitnami/charts/tree/main/bitnami/kube-prometheus
-version: 8.13.1
+version: 8.13.2
```

Comment:
As you are changing the annotations and this could cause some noise to the users, I think it would be better to move to `8.14.0`.

Reply:
Done :)